tests/runtests.py

```diff
@@ -243,7 +243,7 @@ def actual_test_processes(parallel):
 
 
 def django_tests(verbosity, interactive, failfast, keepdb, reverse,
-                 test_labels, debug_sql, parallel):
+                 test_labels, debug_sql, parallel, tags=None, exclude_tags=None):
```

Comment:
I don't see any other way to pass them safely instead of using kwargs? How would you like to do it? I also don't feel uncertain- you can pass any iterable or None to each of these arguments.